docs/_templates/autosummary/class.rst

```diff
@@ -0,0 +1,30 @@
+{{  objname | escape | underline}}
```

Comment:
Did you need to provide this? Or is it for some custom formatting? And if it's for custom, what did you change relative to the defaults?

Reply:
What I'm trying to get at with these questions is, how much of the PR is actually needed to get functioning docs, since we are considering the same framework for napari proper. All the jinja stuff is making my head spin. =)

Reply:
> And if it's for custom, what did you change relative to the defaults?

I changed `{{fullname}}` to `{{objname}}` and I added a conditional to hide the init method